eng/native/init-distro-rid.sh

```diff
@@ -70,8 +70,11 @@ initNonPortableDistroRid()
     fi
 
     if [ "$targetOs" = "FreeBSD" ]; then
-        __freebsd_major_version=$(freebsd-version | { read v; echo "${v%%.*}"; })
-        nonPortableBuildID="freebsd.$__freebsd_major_version-${buildArch}"
+        if (( isPortable == 0 )); then
```

Comment:
I think this condition is not necessary. The comment above already calls it out:

```

# Builds from the following *must* be non-portable:
#
#   |    OS     |           Expected RID            |
#   -------------------------------------------------
#   |  freeBSD  |        freebsd.(version)-x64      |
```

Reply:
I can put in different example (banana Linux?). But last time when we did FreeBSD builds, we used portable freebsd-x64 RID. That is also how source build does it.